painting/src/main/java/com/ataulm/artcollector/painting/ui/PaintingActivity.kt

```diff
@@ -44,8 +45,17 @@ class PaintingActivity : AppCompatActivity() {
             titleArtistTextView.text = "${painting.title} // ${painting.artist.name}" // TODO: string format, but also PaintingUiModel
             creditLineTextView.text = painting.creditLine ?: "Harvard Art Museum"
             creditLineTextView.setOnClickListener { startActivity(webIntent(painting.webUrl)) }
+            informationContainer.visibility = View.VISIBLE
             painting.loadImageIfDifferent()
         })
+
+        imageView.setOnViewTapListener { _, _, _ ->
+            if (informationContainer.visibility == View.VISIBLE) {
+                informationContainer.visibility = View.GONE
```

Comment:
I don't actually want it to behave like this - I would rather the container slide off screen, but it's less important right now.